### PR TITLE
2021 Posts

### PR DESCRIPTION
**Pull request by ntlhui**

### CODE REVIEW

File: _posts/2021-08-04-fishsense-prototype-testing.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    caption="A color (left) and depth (right) image pair"
+    %}
+
+{$ assign image = "assets/fishsense/2021-08-04-grouper-yolov4.png" | resize: "512x288" | absolute_url %}
+{% include img_caption.html
+    src=image
+    caption="Fish detection using YOLOv4"
+    %}
+
+The data we gathered at Birch Aquarium at Scripps also helped us finalize and submit the project's first accepted paper at the OCEANS'21 conference, which takes place right here in San Diego, September 20-23, 2021. You can read the full paper here: [https://kastner.ucsd.edu/wp-content/uploads/2021/08/admin/oceans21-fishsense.pdf](https://kastner.ucsd.edu/wp-content/uploads/2021/08/admin/oceans21-fishsense.pdf).
```
Comment (Sean1572) [inline _posts/2021-08-04-fishsense-prototype-testing.md:34]: https://kastner.ucsd.edu/wp-content/uploads/2021/08/admin/oceans21-fishsense.pdf is missing

**Review by Sean1572 — CHANGES_REQUESTED** [review]

File: _posts/2021-07-20-2021-summer-research-students.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  photo: assets/people/p.tueller.jpg
+---
+This summer, we are hosting 23 students from around the US in our 2021 summer research program, both at UC San Diego and remotely. We'll be working on projects from mapping Maya ruins with LiDaR and mangroves with drone imagery to measuring fish with underwater cameras. Meet our students by reading their bios below:
+
+## Student Researchers
+{% for student in page.students %}
+<div class="floatclearfix">
+    {% if student.photo %}
+    <p><a href="{{ student.photo | absolute_url }}"><img alt="{{student.name}}" class="{{student.align}}" src="{{student.photo | resize: '180x240'}}"></a></p>{% endif %}
```
[the remark this answers is not in the document]
Comment (ntlhui) [reply _posts/2021-07-20-2021-summer-research-students.md]: Fixed in e2121a0bc7cd06fe6c3d20bc9188c99e06f8cf2d

File: _posts/2021-07-20-2021-summer-research-students.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    <p>{{student.bio}}</p>
+</div>
+{% endfor %}
+
+## Team Leads
+{% for student in page.mentors %}
+<div class="floatclearfix">
+    {% if student.photo %}
+    <p><a href="{{ student.photo | absolute_url }}"><img alt="{{student.name}}" class="{{student.align}}" src="{{student.photo | resize: '180x240'}}"></a></p>{% endif %}
```
[the remark this answers is not in the document]
Comment (ntlhui) [reply _posts/2021-07-20-2021-summer-research-students.md]: Fixed in e2121a0bc7cd06fe6c3d20bc9188c99e06f8cf2d

File: _posts/2021-08-04-fishsense-prototype-testing.md (hunk 1 above)
Comment (ntlhui) [reply _posts/2021-08-04-fishsense-prototype-testing.md]: Fixed in e2121a0bc7cd06fe6c3d20bc9188c99e06f8cf2d

**Review by Sean1572 — APPROVED** [review]

**Pull request merged**